app/jsx/layouts/UnitProfileLayout.jsx

```diff
@@ -94,8 +94,9 @@ class UnitProfileLayout extends React.Component {
                        <input type="file" id="logoImage" name="logo" onChange={this.handleImageChange}/>
                        <br/><br/>
                     { this.state.banner_flag_visible &&
-                      [<label key="0" className="c-editable-page__label" htmlFor="logoIsBanner">Suppress typeset site name next to logo: </label>,<p>Check the box below if your logo image contains the full, legible title of your site.</p>,
-                       <input key="1" type="checkbox" id="logoIsBanner" name="logoIsBanner"
+                      [<label key="0" className="c-editable-page__label" htmlFor="logoIsBanner">Suppress typeset site name next to logo: </label>,
+                       <p key="1">Check the box below if your logo image contains the full, legible title of your site.</p>,
+                       <input key="2" type="checkbox" id="logoIsBanner" name="logoIsBanner"
```

Comment:
Note: @justingonder React elements in an array need 'key' attributes defined, each with a unique value.